packages/frontend-2/lib/viewer/composables/ui.ts

```diff
@@ -379,11 +379,23 @@ export function useMeasurementUtilities() {
     state.viewer.instance.getExtension(MeasurementsExtension).clearMeasurements()
   }
 
+  const activeMeasurement = computed(() => {
+    const measurementsExtension =
```

Comment:
I don't think this is actually reactive, cause the viewer certainly isn't. Does it really update in real-time? You can validate this by setting up a watcher on this that emits the value in the console.

If it isn't reactive, better to be clear about that and expose this as a function instead (getActiveMeasurement(): boolean)